packages/react-native-bpk-component-button/src/BpkBorderedButton.ios.js

```diff
@@ -61,6 +60,7 @@ const styles = StyleSheet.create({
 export type Props = {
   backgroundColor: string,
   borderColor: string,
+  borderRadius: number,
```

Comment:
Could this also be a string based on the types 'sm', 'lg', 'pill'? Therefore flow would give an error?